functions/bfc-functions.php

```diff
@@ -535,4 +535,20 @@ function bfc_get_forum_title( $post = 0 ) {
 	return apply_filters( 'bfc_forum_title', $title, $id );
 }
 
+add_filter('bp_core_replace_tokens_in_text', 'bfc_remove_p_in_emails',100);
```

Comment:
Is this properly scoped so it only affects emails? Presumably we do need the HTML sanitization in some places, to protect against script insertion on the site.

Reply:
Agreed. Definitely need HTML sanitization for user generated content that winds up on the website. This function is scoped to just emails via the hook `bp_core_replace_tokens_in_text`. However, thanks to my more recent work on the emails, we don't need this fix any more. It will be removed in an upcoming commit.

Reply:
Nice! 👍